File: NG_empmenu_web/src/main/webapp/js/service/foodsService.js
app.service('foodsService',function ($http) {
    this.findMenuList=function () {
        return $http.get('../foods/findAll.do');
    };

    //增加
    this.add=function (entity) {
        return $http.post('../foods/add.do',entity);
    };

    //查找单个模板对象
    this.findOne=function (menuId) {
        return $http.get('../foods/findOne.do?menuId='+menuId);
    };

    //根据条件查询菜名数据
    this.findPage=function (entity,page,rows) {
        return $http.post('../foods/findPage.do?page='+page+"&rows="+rows,entity);
    }

    //根据选中对象删除
    this.delete=function (selectIds) {
        return $http.get('../foods/delete.do?ids='+selectIds);
    }
});

File: NG_dishes_interface/src/main/java/com/cqm/dishes/service/FoodsService.java
package com.cqm.dishes.service;


import com.cqm.entity.PageResult;
import com.cqm.group.Foods;
import com.cqm.pojo.TbFoods;


public interface FoodsService {
    /**
     * 添加
     * @param foods
     */
    public void add(Foods foods);

    /**
     * 支持分页的指定商家信息信息
     * @param foods
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult findPage(TbFoods foods,int pageNum,int pageSize);

    /**
     * 删除选中的菜名对象
     * @param ids
     */
    public void delete(Long[] ids);

    /**
     * 根据id查询菜名对象
     * @param foodId
     * @return
     */
    public Foods findOne(Long foodId);
}
